content/klc/G2.1.adoc

```diff
@@ -1,21 +1,29 @@
 +++
-brief = "Defining generic and atomic parts"
+brief = "Defining generic and fully specified symbols"
 +++
 
-In general terms, the _symbols_ in the KiCad library can be categorized into two groups:
+KiCad generally has two workflows with regards to assign a footprint to a symbol. It can be done in the library or using the "assign footprints tool" at the end of the design process.
```

Comment:
Remove 'generally'. The number of workflows, at least those defined below, are a known integer number. Change 'with regards to assign' to 'for assigning'. The name of the formerly CvPcb window is just "Assign Footprints" so the word 'tool' can move outside the double quotes. I would add 'schematic' before 'design process'.

Reply:
completely reworded.

Reply:
OK. I only ask to change `"assign footprints tool"` to `"Assign Footprints" tool` for reasons mentioned above.

Reply:
done